ices/66930.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+rustc --crate-type=lib - << EOF
```

Comment:
FWIW you can specify `#![crate_type = "lib"]` in a .rs and it'll be tested as such, not a big enough deal to suggest changing it in this PR though

Reply:
Yeah, we should track `.rs` files basically, fixed.